programmers/완주하지-못한-선수.js

```diff
@@ -0,0 +1,15 @@
+function solution(participant, completion) {
+  const marathon = {};
```

Comment:
이 객체는 참가자가 몇명인지에 대한 객체이기 때문에 이 정도 변수명이 더 적합해 보이네요
```suggestion
  const players = {};
```